api/order/Order.js

```diff
@@ -1 +1,13 @@
-//OrderSchema
+import mongoose from 'mongoose';
+const Schema = mongoose.Schema;
+
+const orderSchema = new Schema({
+  user: {
+    type: Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  email: String,
+  status: String,
```

Comment:
we could use enums here to limit the status options (https://mongoosejs.com/docs/schematypes.html)